docs/developer-docs/frontend/serving-static-assets.mdx

```diff
@@ -0,0 +1,150 @@
+# Serving static assets
+
+## Overview
+
+When a dapp is deployed to ICP, the frontend assets will be displayed via the canister's ID, followed by `.ic0.app`, if the dapp has a frontend configured. For example, the [Motoko Playground](https://m7sm4-2iaaa-aaaab-qabra-cai.ic0.app/) is an application with a frontend user interface that can be used to deploy canisters in a temporary, sandbox environment. This dapp can be accessed via the URL `https://m7sm4-2iaaa-aaaab-qabra-cai.ic0.app/`. 
```

Comment:
Should we also make it more that the canister ID of the Motoko playground is `m7sm4-2iaaa-aaaab-qabra-cai` and therefore, that is why the URL is `https://m7sm4-2iaaa-aaaab-qabra-cai.ic0.app/`?

Reply:
Added clarification for this